lib/hanami/repository.rb

```diff
@@ -208,16 +209,33 @@ module ClassMethods
       #   end
       #
       #   UserRepository.adapter = adapter
+      # @since 0.5.0
+      # @api private
       def adapter=(adapter)
         @adapter = adapter
       end
 
-      # @since 0.5.0
-      # @api private
       def adapter
         @adapter
       end
 
+    end
+
+    module InstanceMethods
+      # Get the currently configured adapter
+      #
+      # Returns the adapter that is specified on the class level
```

Comment:
I do not think we need this comment line since the code reveals the intent well enough